aten/src/ATen/native/ForeachUtils.h

```diff
@@ -128,10 +128,26 @@ inline bool _check_tensors_share_device_and_dtype(
 // corresponding tensors in tensor lists have the same sizes and strides.
 inline bool _check_tensors_share_sizes_and_strides(
     ArrayRef<TensorList> tensorLists) {
+  auto is_same_stride = [](const IntArrayRef& size,
```

Comment:
this looks like it's really
```suggestion
  auto is_diff_stride = [](const IntArrayRef& size,
```
?